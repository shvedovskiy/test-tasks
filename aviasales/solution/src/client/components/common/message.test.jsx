/* eslint-disable no-undef */
import React from 'react';

import Message from './message';


describe('<Message />', () => {
  const message = shallow(<Message>Some Message</Message>);

  it('renders properly', () => {
    expect(message).toMatchSnapshot();
  });
});
