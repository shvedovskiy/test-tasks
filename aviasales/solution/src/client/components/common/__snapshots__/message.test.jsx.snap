// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> renders properly 1`] = `
<message__MessageContainer>
  <message__MessageContent>
    Some Message
  </message__MessageContent>
</message__MessageContainer>
`;
